src/assets/custom/css/_sass/_website-navigation.scss

```diff
@@ -75,6 +75,7 @@
     }
     &.website-navigation__menu--showing-sub-menu {
       left: -100vw;
+      overflow-y: hidden;
```

Comment:
Just be careful that on a small or medium screen that doesn't have sufficient height to show the whole menu you can still scroll it down to reach the views that don't fit on screen

Reply:
Yup, the menus themselves still scroll if they get cut off 😄